client/blocks/inline-help/contextual-help.js

```diff
@@ -905,6 +905,24 @@ const contextLinksForSection = {
 			),
 		},
 	],
+	hosting: [
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/sftp/' ),
+			post_id: 159771,
+			title: translate( 'SFTP Credentials' ),
+			description: translate(
+				"Access and edit your website's files directly by using an SFTP client."
+			),
+		},
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/database-access/' ),
+			post_id: 159822,
+			title: translate( 'Database Access' ),
+			description: translate(
+				'For the tech-savvy, manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/hosting-section-inline-help)